### PR TITLE
Bug:  useRovingKeyIndex RTL direction odd

### LINKED ISSUE #6108: UseRovingKeyIndex RTL direction odd
## Bug Description

This was noticed while implementing `useRovingKeyIndex` on page layouts:
In RTL the left and right arrow keys both go "right/down", and the up and down arrow keys both go "down". So it's like only having two arrow keys rather than four.

See #5989 for more details

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
When WP was in RTL, all arrow keys were navigating to the next sibling in the page layouts tab regardless of direction.

## Summary
This PR just fixes the boolean logic there of determining sibling direction based off of key and RTL. This PR also adds a small unit/regression test for the lil' util used to infer this.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
If you're in RTL mode, and navigate to the page layouts in the editor, now using key navigation should work properly.

## Testing Instructions
Open editor and focus into page layouts library pane. Use arrow keys (up, down, left, right) and see that they navigate properly between page layouts. Switch to RTL and see that the keys still navigate properly (with left & right having reversed functionality from before)

### QA
Follow testing instructions above.

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6108

### CODE REVIEW

**Review by miina — APPROVED** [review]
lgtm!

File: assets/src/edit-story/utils/test/useRovingTabIndex.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * Internal dependencies
+ */
+import { getSiblingDirection } from '../useRovingTabIndex';
+
+describe('getSiblingDirection', () => {
+  describe('RTL', () => {
+    it.each(['ArrowRight', 'ArrowUp', 'PageUp'])(
+      'given key: %s returns "previousSibling"',
+      (key) => {
+        const isRTL = true;
```
Comment (samwhale) [inline assets/src/edit-story/utils/test/useRovingTabIndex.js:26]: nit: the `isRTL` constant can be pulled out of each of these tests and placed in the describe block instead 🤷 not a big deal

File: assets/src/edit-story/utils/useRovingTabIndex.js [hunk 2]
```diff
@@ -67,11 +67,11 @@ function getDistanceSq(p1, p2) {
...
-function getSiblingDirection(isRTL, key) {
-  return !isRTL &&
-    (key === 'ArrowLeft' || key === 'ArrowUp' || key === 'PageUp')
-    ? ['previousSibling']
-    : 'nextSibling';
+export function getSiblingDirection(isRTL, key) {
+  const isPreviousDirection = isRTL
+    ? key === 'ArrowRight' || key === 'ArrowUp' || key === 'PageUp'
+    : key === 'ArrowLeft' || key === 'ArrowUp' || key === 'PageUp';
+  return isPreviousDirection ? 'previousSibling' : 'nextSibling';
```
Comment (samwhale) [inline assets/src/edit-story/utils/useRovingTabIndex.js:74]: ahhhh good find. If it was RTL then it always returned next sibling

**Review by samwhale — APPROVED** [review]
Tested and works great ✨

**Review by BrittanyIRL — APPROVED** [review]
LGTM! Thanks for adding tests! 🚀

**Pull request merged**